wiki/Community/osu!_community_meetings/de.md

```diff
@@ -0,0 +1,43 @@
+
+Vor langer Zeit, in der [fernen Vergangenheit des Spiels](https://twitter.com/ppy/status/1169256824052170755), veranstaltete das osu!-Team regelmäßig Community-Treffen, bei denen jeder willkommen war, um zu reden. Bei der heutigen Größe des Spiels ist dies nicht mehr möglich, aber über Discord wurde eine Art Warteschlangensystem eingeführt, in dem Fragen einzeln beantwortet werden können. Wenn du daran interessiert bist, Fragen zu stellen, stelle bitte sicher, dass Du das unten aufgeführte FAQ durchgelesen hast, nur für den Fall, dass Deine Frage bereits gestellt und beantwortet worden ist!
+
+Derzeit werden die osu! Community-Treffen von [peppy](https://osu.ppy.sh/users/2), [Ephemeral](https://osu.ppy.sh/users/102335) und [smoogipoo](https://osu.ppy.sh/users/1040328) geleitet.
+
+Die Treffen finden in der Regel alle zwei Wochen statt und werden etwa einen Tag vorher auf [peppy's Twitter](https://twitter.com/ppy) angekündigt.
+
+Das nächste osu! Community Meeting findet voraussichtlich am: **22. Januar 2022 (Samstag 21:00 UTC)** statt.
+
+## Sich beteiligen
+
+Die osu! Community Meetings werden derzeit auf einer Discord Stage im [osu!dev Discord](https://discord.gg/ppy) gehostet und auf [peppy's Twitch Kanal](https://www.twitch.tv/ppy) zeitgleich ausgestrahlt. Jedes Treffen dauert voraussichtlich etwa eine Stunde und findet jedes zweite Wochenende statt.  
+
+Jeder ist herzlich eingeladen, mitzumachen! Wenn du eine Frage hast oder etwas zur Diskussion beitragen möchtest, kannst du auf der Discord Stage die Hand heben, um Dich in die Warteschlange einzureihen, oder eine Nachricht in den entsprechenden Discord-Thread schreiben, wenn Du Dich nicht traust zu sprechen, die dann laut vorgelesen und beantwortet wird. Bevor du eine Frage stellst, solltest du außerdem die folgenden Richtlinien beachten:
+
+- Plane deine Gesprächsinhalte im Voraus und halte sie so kurz wie möglich.
+- Versuche nur Themen anzusprechen, die die gesamte Community betreffen, nicht nur dich.
+- Eine Verifizierung auf osu!dev ist nicht notwendig, kann aber deine Chancen erhöhen, für Fragen während der Q&A-Sessions ausgewählt zu werden.
+
+Wir bemühen uns, nicht mehr als 5 Minuten pro Frage zu benötigen. Bitte habe daher Verständnis dafür, dass Fragen, die bereits beantwortet wurden oder besonders allgemein gehalten sind, möglicherweise nicht zur Diskussion gestellt werden.
+
+## Archiv
+
+Das erste osu! Community Meeting fand am 19. September 2021 statt. Alle Treffen wurden und werden auch künftig auf YouTube aufgezeichnet, zusammen mit einem begleitenden zusammenfassenden Dokument.
```

Comment:
```suggestion
Das erste osu! Community Meeting fand am 19. September 2021 statt. Alle Treffen wurden und werden auch künftig auf YouTube aufgezeichnet, zusammen mit einem begleitenden, zusammenfassenden Dokument.
```
I'm pretty sure there needs to be a comma because you can put a `und` between the two adjectives (described at the bottom of [this article by the Duden](https://www.duden.de/sprachwissen/sprachratgeber/Komma-zwischen-Adjektiven))

Reply:
you're right